experimental/BasisLieHighestWeight/src/UserFunctions.jl

```diff
@@ -963,28 +763,21 @@ The monomial ordering is fixed to `degrevlex`.
 ```jldoctest
 julia> mon_bases = basis_coordinate_ring_kodaira_demazure_ffl(:G, 2, [1,0], [1,2], 6)
 6-element Vector{Tuple{MonomialBasis, Vector{ZZMPolyRingElem}}}:
- (Monomial basis of a Demazure module with extremal weight [1, 0] * s1 * s2 over Lie algebra of type G2, [1, x5, x4])
- (Monomial basis of a Demazure module with extremal weight [2, 0] * s1 * s2 over Lie algebra of type G2, [])
- (Monomial basis of a Demazure module with extremal weight [3, 0] * s1 * s2 over Lie algebra of type G2, [])
- (Monomial basis of a Demazure module with extremal weight [4, 0] * s1 * s2 over Lie algebra of type G2, [])
- (Monomial basis of a Demazure module with extremal weight [5, 0] * s1 * s2 over Lie algebra of type G2, [])
- (Monomial basis of a Demazure module with extremal weight [6, 0] * s1 * s2 over Lie algebra of type G2, [])
+ (Monomial basis of a Demazure module with extremal weight (w_1) * s1 * s2 over abstract Lie algebra, [1, x5, x4])
```

Comment:
I would prefer if we could keep the Dynkin type in this output. I would assume that this could be achieved by not using `terse` for the Lie algebra in the oneline printing of the MonomialBasis. Could you please try if that works?